chdb/session/state.py

```diff
@@ -138,9 +138,14 @@ def query(self, sql, fmt="CSV", udf_path=""):
                 - "JSONCompact" - Compact JSON format
                 - "Arrow" - Apache Arrow format
                 - "Parquet" - Parquet format
+                - "DataFrame" - Pandas DataFrame
+                - "ArrowTable" - PyArrow Table
```

Comment:
This part is not directly related to the issue, but I noticed that some formats defined in chdb were not documented in the docstring, so I updated it as well.